crates/swc_ecma_preset_env/src/lib.rs

```diff
@@ -452,8 +464,64 @@ impl VisitMut for Polyfills {
         m.body.retain(|item| !matches!(item, ModuleItem::ModuleDecl(ModuleDecl::Import(ImportDecl { src, .. })) if src.span == DUMMY_SP && src.value == js_word!("")));
     }
 
-    fn visit_mut_script(&mut self, _: &mut Script) {
-        unimplemented!("automatic polyfill for scripts")
+    fn visit_mut_script(&mut self, m: &mut Script) {
+        let span = m.span;
+        let required = self.collect(m);
+        if cfg!(debug_assertions) {
```

Comment:
What's the purpose of this?

Reply:
I don't known, but visit_mut_module is written like this.